configs/vision/simplecnn_mnist.yaml

```diff
@@ -19,30 +20,33 @@ training:
     terminal_conditions:
         loss_stop: 1.0e-5
         episode_limit: 20000
-        epoch_limit: 10
+        epoch_limit: 1
 
 # Problem parameters:
 validation:
     problem:
         name: *name
-        batch_size: *b
+        batch_size: 5000
```

Comment:
You specify the use of `SubsetRandomSampler` below, so a more 'regular' batch size could be used, no?